src/wizard/__tests__/wizard.test.tsx

```diff
@@ -216,7 +218,7 @@ describe('Primary button', () => {
       if (index < DEFAULT_STEPS.length - 1) {
         expect(wrapper.findPrimaryButton().getElement()).toHaveTextContent(DEFAULT_I18N_SETS[0].nextButton!);
       } else {
-        expect(wrapper.findPrimaryButton().getElement()).toHaveTextContent(DEFAULT_I18N_SETS[0].submitButton);
+        expect(wrapper.findPrimaryButton().getElement()).toHaveTextContent(DEFAULT_I18N_SETS[0].submitButton!);
```

Comment:
Since `renderDefaultWizard` uses the new `submitButtonText` prop, the case of using `i18nStrings.submitButton` is not covered now. This assertion passes because the value of both is the same. Since using the old prop should still be supported even if it's deprecated, I think we should keep covering this case in tests to make sure our fallback logic is not broken.